pkg/pillar/Dockerfile

```diff
@@ -1,8 +1,8 @@
 # Copyright (c) 2018 Zededa, Inc.
 # SPDX-License-Identifier: Apache-2.0
-FROM lfedge/eve-alpine:3a7658b4168bcf40dfbcb15fbae8979d81efb6f1 as build
+FROM lfedge/eve-alpine:7.8.0 as build
```

Comment:
Nothing wrong with moving to a proper release in pillar, but let's not mix these two. This and its dependencies should be its own PR.